ui/pages/token-allowance/token-allowance.js

```diff
@@ -347,22 +348,18 @@ export default function TokenAllowance({
             name={origin}
             url={siteImage}
           />
-          <Typography
-            variant={TypographyVariant.H6}
-            fontWeight={FONT_WEIGHT.NORMAL}
+          <Text
+            variant={TextVariant.bodySm}
+            as="h6"
             color={TextColor.textAlternative}
             boxProps={{ marginLeft: 1, marginTop: 2 }}
```

Comment:
`boxProps` is not a valid prop for the `Text `component. Instead, you can pass the `margin` prop directly. Also, I don't think we need `marginTop` here since it's not required. Refer to the ss

```suggestion
            as="h6"
            color={TextColor.textAlternative}
            marginLeft={1}
```
![Screenshot 2023-05-02 at 3 33 21 PM](https://user-images.githubusercontent.com/39872794/235638182-7d903b08-2a73-4681-8dd1-286f84abe3b4.jpg)